### PR TITLE
chore: Support @cloudscape-design npm-style imports in sass

### PR DESCRIPTION
**Pull request by gethinwebster**
*Issue #, if available:*

*Description of changes:*

Allow e.g. `@use '@cloudscape-design/component-toolkit/internal/focus-visible' as focus-visible;` in SASS files.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by Al-Dani — APPROVED** [review]

File: package-lock.json [hunk 1]
```diff
@@ -931,6 +932,15 @@
         "webdriverio": "^7.25.2"
       }
     },
+    "node_modules/@cloudscape-design/component-toolkit": {
```
Comment (just-boris) [inline package-lock.json:935]: Why did the lock clean script not work here?

**Review by just-boris — COMMENTED** [review]
How about adding a test for this change?

File: package.json [hunk 2]
```diff
@@ -23,6 +23,7 @@
   "dependencies": {
     "@babel/plugin-transform-modules-commonjs": "^7.18.6",
     "@cloudscape-design/browser-test-tools": "^3.0.2",
+    "@cloudscape-design/component-toolkit": "^1.0.0-beta.53",
```
Comment (just-boris) [inline package.json:26]: Should this be a dev-dependency?

**Comment by gethinwebster** [comment]
> How about adding a test for this change?

https://github.com/cloudscape-design/theming-core/pull/80/files#diff-fabf997d5608c1391298eb6a21c84b6b18fa541da0a36985e2bf53853937dbea

**Comment by just-boris** [comment]
ok, found the test, but the dependency management questions still stand

File: package.json (hunk 2 above)
Comment (gethinwebster) [reply package.json:26]: will update - have another PR coming up anyway as this needed some tweaks after further end-to-end testing

**Pull request merged**